### PR TITLE
Initial commit with article about skunk and github actions

### PR DESCRIPTION
**Pull request by etagwerker**
Hey @lubc, 

This is my new article about skunk and GitHub actions. 

Please check it out. 

Thanks!

### CODE REVIEW

File: _posts/2020-02-10-calculate-tech-debt-using-skunk-on-github-actions.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+Feature branch ((HEAD detached at 0315f34)) average stink score: 13.42
+Score: 13.42
+```
+
+This particular example shows that there is no difference in my application's
+technical debt: [https://github.com/fastruby/skunk/pull/26](https://github.com/fastruby/skunk/pull/26).
+That's because I didn't change any Ruby code (just a YAML file).
+
+You can see the GitHub Action run over here:
+[https://github.com/fastruby/skunk/runs/437118684](https://github.com/fastruby/skunk/runs/437118684)
```
Comment (lubc) [inline _posts/2020-02-10-calculate-tech-debt-using-skunk-on-github-actions.md]: @etagwerker fyi, it shows a syntax error there
```
running simple_cov
...............
sh: 1: Syntax error: "(" unexpected
```

File: _posts/2020-02-10-calculate-tech-debt-using-skunk-on-github-actions.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
+---
+layout: post
+title: "How to Calculate Tech Debt Using Skunk on GitHub Actions"
+date: 2020-02-10 16:16:00
+categories: ["tech-debt"]
```
Comment (lubc) [inline _posts/2020-02-10-calculate-tech-debt-using-skunk-on-github-actions.md]: @etagwerker let's add `code-quality` too. Your other articles about Skunk are under that category

File: _posts/2020-02-10-calculate-tech-debt-using-skunk-on-github-actions.md (hunk 1 above)
Comment (etagwerker) [reply _posts/2020-02-10-calculate-tech-debt-using-skunk-on-github-actions.md]: @lubc Yes, I noticed that, but I don't think it is a big issue.

Comment (etagwerker) [reply _posts/2020-02-10-calculate-tech-debt-using-skunk-on-github-actions.md]: It's been reported here: https://github.com/whitesmith/rubycritic/issues/346

**Comment by etagwerker** [comment]
@lubc Just addressed your comments. 👍

**Pull request merged**